dist/index.html

```diff
@@ -4,31 +4,92 @@
     <meta charset="utf-8">
     <title>Seek-A-Design Playground</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7fcb54df80.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <script src="../src/index.js"></script>
   </head>
   <body>
     <div class="parent-container">
-      <!-- <h1 class="center">Hello World!</h1> -->
       <div class="navigation-container">
-        <button type="button" class="nav">Home</button>
-        <button type="button" class="nav">Projects</button>
-        <button type="button" class="nav">About Me</button>
-        <button type="button" class="nav">Contact</button>
+        <button type="button" class="nav-home">
+            <i class="fas fa-home"></i> Home
```

Comment:
**Lines 15, 18, 21, and 24:**

Careful! You're over-indenting by two spaces here.